test/realtime/trip_modification_test.exs

```diff
@@ -15,7 +15,17 @@ defmodule Realtime.TripModificationTest do
       build(:gtfs_stop, id: "1236")
     ]
 
-    shape_with_stops = build(:shape_with_stops, id: "id-of-the-shape")
+    shape_with_stops =
+      build(:shape_with_stops,
+        id: "id-of-the-shape",
+        stops: [
+          build(:gtfs_stop, id: "1233"),
```

Comment:
This was moderately annoying - I needed to do this because the stops all came out of the factory with the same ID otherwise.

Eventually it may be worth looking into whether we could make the factories do a bit of randomizing for us, but I figured this was fine for now.

Reply:
Updating the factory for this is fairly easy, we can pair on it Monday? Or I can give you a diff/PR